packages/moleculer-db/moleculer-db.d.ts

```diff
@@ -1,22 +1,24 @@
 declare module "moleculer-db" {
   import { Context, ServiceBroker, Service } from "moleculer";
 
+  export interface DbServicePopulate extends Record<string,any>{
+     <R>(ids: string|number, docs: any[], rule: any, ctx: Context): Promise<R>;
+  }
   export interface DbServiceSettings {
     idField: string;
     fields?: Array<string>;
-    populates?: Array<any>;
+    populates?: Array<DbServicePopulate>;
     pageSize?: number;
     maxPageSize?: number;
     maxLimit?: number;
-    entityValidator?: any;
+    entityValidator?: <T, R>(entity: T)=> Promise<R>;
   }
-
-  export default class DbService<T=any,S extends DbServiceSettings = DbServiceSettings> extends Service<S> {
+ export interface DbEntity<T extends Record<string,any> = Record<string,any>> extends Record<string,any> {}
```

Comment:
The only use of `DbEntity` appears to be as a constraint for the generic in `DbAdapter` which I don't see being used (other comment).  If this is needed, please clean up the formatting a bit (indentation, spacing between interface definitions).